Arduino/CN470RP1/TransmissionTestABPLoRa/TransmissionTestABPLoRa.ino

```diff
@@ -0,0 +1,111 @@
+
+#include <LoRaWan.h>
+// THOSE ARDUINO SAMPLE REQUIRES CHANGES IN THE ARDUINO CODE TO RUN DESCRIBED IN THE MAIN REPO
+// THE CONCENTRATOR IS EXPECTED TO HAVE THE FOLLOWING FREQUENCIES ACTIVATED
+// Cn470 Channels 1-4 : 470.3, 470.5, 470.7, 470.9
+// Cn470 Channels 39-42: 478.1, 478.3, 478.5, 478.7
+// please refer to the repo documentation for further information
+//set to true to send confirmed data up messages
+bool confirmed = true;
+//application information, should be similar to what was provisiionned in the device twins
+char * deviceId = "46AAC86800430028";
+char * devAddr = "0228B1B1";
+char * appSKey = "2B7E151628AED2A6ABF7158809CF4F3C";
+char * nwkSKey = "3B7E151628AED2A6ABF7158809CF4F3C";
+
+
+/*
+  iot hub ABP tags for deviceid: 46AAC86800430028
+    "desired": {
+    "AppSKey": "2B7E151628AED2A6ABF7158809CF4F3C",
+    "NwkSKey": "3B7E151628AED2A6ABF7158809CF4F3C",
+    "DevAddr": "0228B1B1",
+    "GatewayID" :"",
+    "SensorDecoder" :"DecoderValueSensor"
+    },
+*/
+
+//set initial datarate and physical information for the device
+_data_rate_t dr = DR6;
+_physical_type_t physicalType = EU868 ;
+
+//internal variables
+char data[10];
+char buffer[256];
+int i = 0;
+int lastCall = 0;
+
+
+void setup(void)
+{
+  SerialUSB.begin(115200);
+  while (!SerialUSB);
+  lora.init();
+  lora.setDeviceDefault();
+
+  lora.setId(devAddr, deviceId, NULL);
+  lora.setKey(nwkSKey, appSKey, NULL);
+
+  lora.setDeciveMode(LWABP);
+  lora.setDataRate(dr, physicalType);
+
+  // Disable Channels not used from the active channel list
+  for (int i = 4 ; i< 39; i++){
+    lora.setChannelOFF(i);
```

Comment:
@Mandur `setChannelOFF()` doesn't seem to compile correctly, maybe we missed an import?

C:\Users\masalak\code\iotedge-lorawan-starterkit\Arduino\CN470RP1\TransmissionTestABPLoRa\TransmissionTestABPLoRa.ino: In function 'void setup()':
TransmissionTestABPLoRa:53:10: error: 'class LoRaWanClass' has no member named 'setChannelOFF'; did you mean 'setChannel'?
     lora.setChannelOFF(i);
          ^~~~~~~~~~~~~
          setChannel
TransmissionTestABPLoRa:56:10: error: 'class LoRaWanClass' has no member named 'setChannelOFF'; did you mean 'setChannel'?
     lora.setChannelOFF(i);
          ^~~~~~~~~~~~~
          setChannel
exit status 1
'class LoRaWanClass' has no member named 'setChannelOFF'; did you mean 'setChannel'?